src/Frame.jl

```diff
@@ -127,14 +151,18 @@ end
 
 
 """
-Guess the bonds, angles and dihedrals in the ``frame`` using a distance criteria.
+    guess_bonds!(frame::Frame)
+
+Guess the bonds, angles and dihedrals in the `frame` using a distance criteria.
```

Comment:
Nit pick: bonds, angles **,** and dihedrals ...
(technically, this comma is optional - but typically recommended).

Reply:
I prefer having it =)